### PR TITLE
Adding weather variables using "current" parameter

### PR DESCRIPTION
**Pull request by hundredfire**
As discussed, I'm trying to add new weather variables to the API call. Instead of using the current_weather parameter, I'm using "current" like in the API call examples.
I added the "precipitation", relative_humidity_2m" and "weather_code" variables to the model.go to add to the Response struct, and modified the exporter.go accordingly to use these, and added some basic descriptions. 

Full-disclosure: This is probably completely wrong as I've never used Go (I have basically no coding experience, although I'm interested in learning) and **asked ChatGPT for help**, but I'm using this PR to discuss how I could help implement this and as a learning opportunity. If this is inappropriate, please disregard and discard this. Thanks again!

### CODE REVIEW

**Comment by hundredfire** [comment]
I forgot to add the example API call I'm basing this off:
`https://api.open-meteo.com/v1/forecast?latitude=43.7031&longitude=7.2661&current=temperature_2m,relative_humidity_2m,precipitation,weather_code,wind_speed_10m,wind_direction_10m&timezone=Europe%2FBerlin&forecast_days=1`

File: .github/workflows/docker-image.yml
Comment (rkosegi) [inline .github/workflows/docker-image.yml:1]: There already is a workflow to build/publish container image

Comment (hundredfire) [reply .github/workflows/docker-image.yml:1]: my bad. I was trying to build an image to test on my setup. I deleted the docker-image.yml

**Comment by hundredfire** [comment]
Ok, I think I'm way out of my depth here. Sorry, I tried, but I can't seem to be able to make the build work again...

**Pull request closed without merging**